docs/modules/ROOT/pages/tutorials/installation.adoc

```diff
@@ -0,0 +1,83 @@
+= Installation
+
+This tutorial goes through the steps required to get the operator up and running.
+
+== Prerequisites
+
+* `kubectl`
+* `helm` v3
+* `yq`
+* `base64`
+* `openssl`
+* A running Kubernetes cluster with cluster-admin permissions
+
+== Steps
+
+. Install Crossplane
++
+[source,bash]
+----
+helm repo add crossplane https://charts.crossplane.io/stable
+helm upgrade --install crossplane crossplane/crossplane \
+  --create-namespace --namespace crossplane-system \
+  --wait
+----
+
+. Install provider-helm
++
+[source,yaml]
+----
+kubectl apply -f - << EOF
+include::example$installation/rbac.yaml[]
+EOF
+
+kubectl apply -f - << EOF
+include::example$installation/controller-config.yaml[]
+EOF
+
+kubectl apply -f - << EOF
+include::example$installation/provider-helm.yaml[]
+EOF
+
+kubectl apply -f - << EOF
+include::example$installation/providerconfig-helm.yaml[]
+EOF
+
+kubectl wait --for condition=Healthy provider.pkg.crossplane.io/provider-helm --timeout 60s
```

Comment:
Works for me with `provider.pkg.crossplane.io`.

Reply:
It would be nice if you provide your output as requested before, so that I can help resolve the problem :)
Otherwise, I'll ignore your comments since Simon confirmed it works :shrug: